srcpkgs/mozjs60/template

```diff
@@ -46,6 +46,12 @@ do_check() {
 	dist/bin/jsapi-tests
 }
 
+post_install() {
+	# Fix the '-include' directive, otherwise it tries to use the hosts' header
+	vsed 's|^Cflags:.*|Cflags: -include ${pc_sysrootdir}/${includedir}/mozjs-60/js/RequiredDefines.h -I${includedir}/mozjs-60|' \
+		-i ${DESTDIR}/usr/lib/pkgconfig/mozjs-60.pc
```

Comment:
This doesn't seem to break cross/native compilation, but this still seems pretty hacky to me somehow.

Reply:
It is using documented features of pkg-config